### DISCUSSION TITLE
Where did "Show icons hierarchically" go?

### DISCUSSION THREAD

**Discussion by martinschweikert** [freeplane/freeplane#1220] [Help]
I noticed that the menu item "View > View settings > Show icons hierarchically" is no longer there - where did it go?

I found a hint about "Edit > Icons", but that is also not there; must have been part of an older version.

![Qf2r4WjFE5](https://github.com/freeplane/freeplane/assets/84708136/21febdb9-d4d9-47f0-8495-e05b1a019348)

Best regards,
Martin

**Comment by dpolivaev** [comment]
<img width="405" alt="searchDialog" src="https://github.com/freeplane/freeplane/assets/1558052/31930991-9c24-4897-8830-0d844aa9da40">

Control + F1 shows you the command search dialog

**Reply by martinschweikert** [reply]
Great!

Thank you
1) for pointing me to the menu item
2) for the hint to the command search dialog. I am using Freeplane (and before that: Freemind) for many years now, but this one I did not know!

Thank you!
